### PR TITLE
add Ukrainian translation

### PR DESCRIPTION
**Pull request by JekRock**
### Changes
Add Ukrainian translation

### Testing
- [x] Tested on a physical device
- [ ] Added or modified unit test cases

### Others
- Fixes #0
- Connects #200

### CODE REVIEW

**Comment by JekRock** [comment]
Also, there are a few issues I've found on a real device.
Here the text is too long to fit and it's not scrollable.

Here and on the previous screenshot, the "skip" and "done" buttons aren't translated for some reason. They aren't present in the resources list though.

**Review by ashutoshgngwr — CHANGES_REQUESTED** [review]
Hey @JekRock, thank you for taking the time to create this PR. I found 2 issues, please take a look at them.

Regarding the concerns that you pointed out:

> Also, there are a few issues I've found on a real device. Here the text is too long to fit and it's not scrollable.

You can rephrase the text to be shorter. User will tend to skip very long texts in the intro slides. The original intent of the message was to explain what the time slider does.

> Here and on the previous screenshot, the "skip" and "done" buttons aren't translated for some reason. They aren't present in the resources list though.

These are provided by the [AppIntro](https://github.com/AppIntro/AppIntro) library. You can create a PR in their repository to provide Ukrainian translations. See [this](https://github.com/AppIntro/AppIntro#translating-).

File: app/src/main/res/values-uk-rUA/strings.xml [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+  <string name="seaside">Морський берег</string>
+  <string name="soft_wind">Легкий вітер</string>
+  <string name="thunder_crack">Тріск грому</string>
+  <string name="train_horn">Гудок поїзду</string>
+  <string name="walking_through_the_snow">Прогулянка по снігу</string>
+  <string name="water_hose">Водяний шланг</string>
+  <string name="water_stream">Потік води</string>
+  <string name="white_noise">Білий шум</string>
+  <string name="wind_in_palm_trees">Вітер у пальмах</string>
+  <string name="wind_in_chimes_of_shells">Китайський дзвіночок із мушель</string>
```
[the remark this answers is not in the document]
Comment (JekRock) [reply app/src/main/res/values-uk-rUA/strings.xml:50]: It was the most difficult to translate.
If I understand it correctly, it's the sound made by something that looks similar to [that](https://duckduckgo.com/?t=ffab&q=chime+of+shells&iax=images&ia=images&iai=http%3A%2F%2Fwww.quiet-corner.com%2Fwp-content%2Fuploads%2F2016%2F07%2FGreat-DIY-Seashell-Wind-Chimes-Ideas-4.jpg)
There is no short name for that thing in Ukrainian and the closest description is what I put into translation.
Russian translation has the word "Фурин" ([link](https://github.com/ashutoshgngwr/noice/blob/44fd8c85fb249727e43e7a1567b197bb5d3109ae/app/src/main/res/values-ru/strings.xml#L39)) that means "Japanese bell" (I don't agree with that translation).

I can change it to "Дзвін мушель" (bell of shells) if you think that it's a better translation for that.

File: fastlane/metadata/android/uk-UA/short_description.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+Розслабтесь, покращить фокус та збільште продуктивність з мінімальним фоновим шумом
```
[the remark this answers is not in the document]
Comment (JekRock) [reply fastlane/metadata/android/uk-UA/short_description.txt:1]: Sure, I'll rephrase the description to make it fit the 80 characters limitation

**Comment by JekRock** [comment]
> You can rephrase the text to be shorter. User will tend to skip very long texts in the intro slides. The original intent of the message was to explain what the time slider does.
Got it, I'll rephrase it

> These are provided by the AppIntro library. You can create a PR in their repository to provide Ukrainian translations. See this
Thanks, I'll send them a PR with translation

**Comment by JekRock** [comment]
Also, I can help with updating the Russian translation, if you don't mind

File: app/src/main/res/values-uk-rUA/strings.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (ashutoshgngwr) [reply app/src/main/res/values-uk-rUA/strings.xml:50]: @JekRock Can we make it _Bells of glass shells_?

**Comment by ashutoshgngwr** [comment]
@JekRock Thanks for the quick response.

> Also, I can help with updating the Russian translation, if you don't mind

Yes, please. Translations are lagging behind by a lot.

File: app/src/main/res/values-uk-rUA/strings.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (JekRock) [reply app/src/main/res/values-uk-rUA/strings.xml:50]: I'm not sure that it sounds like _glass_ shells (I may be wrong).
But generally, yes, it's kind of similar to what you suggested.

**Comment by JekRock** [comment]
> Yes, please. Translations are lagging behind by a lot.

Sure, I'll send a PR after we finish with the Ukrainian translation

**Comment by JekRock** [comment]
Also, do you have a template for the featureGraphic?
I would like to add it to the translation as well.

**Comment by ashutoshgngwr** [comment]
@JekRock I don't have a template. I create it at canva.com using a natural image for background. The dimensions must be 1024x500.

File: app/src/main/res/values-uk-rUA/strings.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (ashutoshgngwr) [reply app/src/main/res/values-uk-rUA/strings.xml:50]: Yeah right. More like metal shells. _Bell of shells_ is fine I guess!

**Comment by JekRock** [comment]
@ashutoshgngwr I've made changes that we've discussed.
Also, I've added the featureGraphic for the Ukrainian translation. Let me know if it looks fine.

**Comment by JekRock** [comment]
I would appriciate if you could add the "hacktoberfest-accepted" label to the PR before merging. If not, that's fine, no worries

**Comment by ashutoshgngwr** [comment]
@JekRock I've added the Hacktoberfest topic to the repository so I don't think it'll be necessary but I'll add it anyways.

**Review by ashutoshgngwr — APPROVED** [review]

**Comment by JekRock** [comment]
@ashutoshgngwr Thank you!

**Pull request merged**